_includes/footer.html

```diff
@@ -64,6 +65,7 @@
             src="{{site.baseurl}}/assets/img/fontawesome-free-5.11.2-web/svgs/brands/vimeo-v.svg"
             alt="Vimeo"
             class="brand-icon"
+	    aria-label="Vimeo"
```

Comment:
This is already covered by [the `alt` attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Img#Attributes).

```suggestion
```